synthesizer/src/coinbase_puzzle/mod.rs

```diff
@@ -310,6 +323,15 @@ impl<N: Network> CoinbasePuzzle<N> {
         };
 
         // Compute the accumulator evaluation.
+        #[cfg(feature = "serial")]
+        let mut accumulator_evaluation = cfg_iter!(prover_polynomials).zip_eq(&challenge_points).fold(
+            <N::PairingCurve as PairingEngine>::Fr::zero(),
+            |accumulator, (prover_polynomial, challenge_point)| {
+                accumulator + (prover_polynomial.evaluate(accumulator_point) * challenge_point)
+            },
+        );
+
+        #[cfg(not(feature = "serial"))]
```

Comment:
@Pratyush can you check if there is a more elegant/integrated approach here?

Reply:
Same suggestion as above.